frontend-site/src/modules/staticData.js

```diff
@@ -1,29 +1,72 @@
 export const utilitiesOffered = Object.freeze([
+  'Hardware',
+  '3-D Printing',
+  '3-D Scanning',
   'Raspberry Pi',
   'Arduino',
+  'LUG API',
+  'Board Games',
+  'Game Systems',
   'Ice Cream',
-  '3-D Printing',
 ]);
 
 export const servicesOffered = Object.freeze([
   // TODO: fill in empty entries
   {
-    name: 'Raspberry Pi',
-    image: '',
-    description: '',
+    name: 'Hardware',
+    image: require("@/assets/Hardware.png"),
+    description: 'From resistors to soldering irons, the LUG is stocked with a wide variety of hardware componants available for student use.',
     url: '',
   },
+  {
+    name: '3D Printing',
+    image: require("@/assets/3DPrinter.png"),
+    description: 'Becoming a LUG member brings many perks such as having easy access to our 3D printer located in the ACM/LUG office. Members can watch their creations come alive for minimal cost.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
+  {
+    name: '3-D Scanning',
+    image: require("@/assets/3DScanner.png"),
+    description: 'A 3-D scanner is a device that scans and analyses objects and returns a 3-D 360° image. Available for use in the LUG office, students will have access to a rare 3-D technology, which could be used along side the 3-D scanner.',
+    url: 'https://lug.cs.uic.edu/3dprint/home',
+  },
+  {
+    name: 'Raspberry Pi',
+    image: require("@/assets/RasberryPi.jpg"),
+    description: 'The Raspberry Pi, single-board computer, has been instrumental in encouraging student creativity and LUG is here to help students gain access to one. We offer Raspberry Pi rentals for any student interested. A Raspberry Pi is powerful enough to run a full Linux installation and many student projects; ask an officer if it\'s the right choice for you.',
+    url: '',
+    },
   {
     name: 'Arduino',
-    image: '',
-    description: '',
+    image: require("@/assets/Arduino.jpg"),
+    description: 'Arduino was born to be an easy tool for fast prototyping, aimed at students without a background in electronics and programming. If you are in need of an Arduino for a class or simply want to test out the technology, LUG has you covered.',
     url: '',
   },
   {
-    name: '3D Printing',
-    image: '',
-    description: '',
-    url: 'https://lug.cs.uic.edu/3dprint/home',
+    name: 'LUG API',
+    image: require("@/assets/API.png"),
+    description: 'LUG is determined to present students with tools that will help them on their road to success, which is why we made our API open to anyone, presenting students with the opportunity to see how this technology works.',
+    url: '',
   },
+  {
+    name: 'Board Games',
+    image: require("@/assets/BoardG.jpg"),
+    description: 'Need a break a stressful week of classes? Pick up a board game, located in the LUG office, and grab some friends. There are plenty of games to choose from.',
+    url: '',
+  },
+  {
+    name: 'Game Systems',
+    image: require("@/assets/Sega.png"),
```

Comment:
Please compress your images. Running the site lags a lot with loading such high resolution images. I don't think we'll need a width higher than 150 or 200 pixels, but I'll leave it up to your judgement.

![image](https://user-images.githubusercontent.com/16447522/49127068-92149b00-f28b-11e8-9b9e-a44f5769d7c5.png)

Reply:
If it works for you, Ill just make them all JPG files.

Reply:
You don't necessarily have to convert them to JPG, just lowering the resolution should be enough.

For example, I resized the Sega image to be only 200px wide (keeping the same aspect ratio). You can see the size difference below. (Only PoC, not committed at time of writing)

![image](https://user-images.githubusercontent.com/16447522/49151128-afba2280-f2d4-11e8-8a7d-edde8f85c719.png)

And here's what it looks like with the resized Sega image. I don't see any immediate difference in quality, and we can scrutinize the quality of a lower resolution together if you want.

![image](https://user-images.githubusercontent.com/16447522/49151067-72559500-f2d4-11e8-8619-a102afdcac84.png)


High resolution images are nice to have (and do tend to look nice on large displays), but I don't think we need to load 100+MB of 4k images for what will be probably 200px to 500px wide at most on the page. If you want, we can keep the high resolution images, but I don't think it's a good idea to load them by default (considering limited data plans and page performance).